_i18n/es.yml

```diff
@@ -85,387 +85,387 @@ home:
-  cryptonote: Cryptonote Whitepapers
-  cryptonote-whitepaper: Cryptonote Whitepaper
-  cryptonote-whitepaper_para: This is the original cryptonote paper written by the cryptonote team. Reading it will give an understanding about how the cryptonote algorithm works in general.
-  annotated: Annotated Whitepaper
-  annotated_para: The Monero Research Lab released an annotated version of the cryptonote whitepaper. This is sort of like an informal review of the claims that are made line-by-line of the whitepaper. It also explains some of the harder concepts in relatively easy to understand terms.
-  brandon: Brandon Goodell's Whitepaper Review
-  brandon_para: This paper is a formal review of the original cryptonote paper by MRL researcher Brandon Goodell. He takes an in-depth look at the claims and mathematics presented in the cryptonote paper.
+  translated: "yes"
+  intro: Monero no está comprometido solamente a hacer una moneda fungible, sino también a una continua investigación en el reino de la privacidad financiera ya que esto implica una criptomoneda. A continuación encontrarás el trabajo de nuestro propio Laboratorio de Investigación de Monero, con más trabajos por delante.
+  mrl_papers: Papeles del Laboratorio de Investigación de Monero (en inglés)
+  abstract: Abstracto
+  introduction: Introducción
+  read-paper: Leer el papel
+  mrl1: Una nota en Reacciones en Cadena en Trazabilidad en CryptoNote 2.0
+  mrl1_abstract: Este boletín de investigación describe un ataque plausible en un sistema de anonimato basado en firma de círculo. Utilizamos como motivación el protocolo de criptomoneda CryptoNote 2.0 aparentemente publicado por Nicolas van Saberhagen en 2012. Ha sido previamente demostrado que la imposibilidad de rastrear un par de llaves de un solo uso pue
```

Comment:
"See the"

"Ver la"
No estoy seguro que querra decir en la página.

Reply:
Hoy estaré realizando algunas de las correcciones que compartiste @lh1008 gracias por tomarte el tiempo de revisar todo el documento :)

Reply:
A ti por aceptar los comentarios y gracias a ti por tu contribución a la comunidad.